webrtc-stats.html

```diff
@@ -1298,6 +1299,17 @@ <h2>
                   the first frame is decoded this attribute is missing.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>frameBitDepth</code></dfn> of type <span class="idlMemberType">unsigned
+                long</span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for video. Represents the bit depth per pixel of the raw video frame.
```

Comment:
Change this to of the encoded frame instead of raw. And add another metric on the media-source for the raw bit depth.

Reply:
The idea would be if 888 is the raw, the encoder encodes it to 444 and sends it, then 444 is received and renders it to 888 or 444. in this case, I'd like to see the original 888, the encoded 444, and the final decoded 888 or 444

Reply:
For consistency with my other comment, I suggest rephrasing it:
"Represents the bit depth per pixel of the last decoded frame"

Reply:
Just for fun ... are you reporting the bit depth for one color, or the bit depth for a pixel?
in 4.2.0 the colors have different bit depths - I suppose this would be the "encoded" bit depth.

Reply:
@alvestrand Thanks for the correction! I just saw your first message in my mail and have been thinking about whether this was the case or not.

Reply:
@vr000m Thanks for the suggestions!
However, I'm a bit confused here, what do you mean by bit depth of the encoded video frame? To me it only makes sense to talk about bit depth when the frame is not encoded, i.e., raw. When the frame is encoded, the bits are stored in a buffer that only has a meaning to the decoder. The only meaning of a bit depth field for the encoded video frame would be to determine the bit depth of the frame after it's been decoded.
What do you think of adding frameBitDepth to RTCVideoSourceStats as suggested, but removing it completely from RTCOutboundRtpStreamStats?

Reply:
I was trying to figure out where or which dictionary the metric should be reported on. The discussion with @henbos was if this is before encoding and after. Also if the bit depth can change. 

If this is a value before encoding then outbound would be the wrong place to put it.

Reply:
Thanks, then I understand.
I've removed it from outbound and added it to RTCVideoSourceStats.